packages/iov-ethereum/src/ethereumconnection.spec.ts

```diff
@@ -795,7 +772,124 @@ describe("EthereumConnection", () => {
       }
 
       connection.disconnect();
-    }, 50_000);
+    }, 60_000);
+
+    it("lists ERC20 transactions when searching by recipient", async () => {
+      pendingWithoutEthereum();
+
+      const connection = await EthereumConnection.establish(testConfig.base, {
+        ...testConfig.connectionOptions,
+        erc20Tokens: testConfig.erc20Tokens,
+      });
+      const codec = new EthereumCodec({ erc20Tokens: testConfig.erc20Tokens });
+
+      const profile = new UserProfile();
+      const wallet = profile.addWallet(Secp256k1HdWallet.fromMnemonic(testConfig.mnemonic));
+      const mainIdentity = await profile.createIdentity(wallet.id, testConfig.chainId, HdPaths.ethereum(0));
+
+      const recipient = await randomAddress();
+
+      for (const sendTest of testConfig.erc20TransferTests) {
```

Comment:
I find it odd that we're combining configuration options for a test context with test data in this way.

Reply:
Me too. But I do not have a great idea how to change that. I mean we can enforce a list length of 2 and can have a test for the primary and a test for the secondary token. Not sure how that scales.

Reply:
But what I find worse the current version is creating tests dynamically (call `describe` or `it` based on runtime data). We do this in iov-tendermint-rpc and it is very hard to follow the test code.